src/init.cpp

```diff
@@ -1728,6 +1728,7 @@ bool AppInitMain(InitInterfaces& interfaces)
     connOptions.nMaxOutboundLimit = nMaxOutboundLimit;
     connOptions.m_peer_connect_timeout = peer_connect_timeout;
     connOptions.m_listen_port = listen_port;
+    connOptions.m_default_outbound_port = chainparams.GetDefaultPort();
```

Comment:
In commit "net: pass in default outgoing port" (f01ebc1a8f0a1e5845c90d6a156ccc05b2c366eb):

I'd maybe `s/outbound_port/connect_port/` in this commit. I could be off, but  "outbound" to me suggests the port connected _from_ not the port connecting _to_.